serde-support/src/main/java/io/micronaut/serde/support/deserializers/PropertiesBag.java

```diff
@@ -52,6 +61,44 @@ private PropertiesBag(BeanIntrospection<T> beanIntrospection,
         this.originalNameToPropertiesMapping = originalNameToPropertiesMapping;
         this.properties = properties;
         this.nameToPropertiesMapping = nameToPropertiesMapping;
+        if (properties.length > 0 && properties.length <= 64) {
+            this.propertiesMask = -1L >>> (64 - properties.length);
+        } else {
+            this.propertiesMask = 0;
+        }
+        Stream<String> propStream = beanIntrospection.getBeanProperties().stream().map(Named::getName);
+        if (nameToPropertiesMapping != null) {
+            propStream = Stream.concat(propStream, nameToPropertiesMapping.keySet().stream());
+        }
+        Set<String> props = propStream.collect(Collectors.toSet());
```

Comment:
This is not needed anymore after the changes in Core

Reply:
i think it should stay, because propertyIndexOfSlow still does a bit more work than just calling the core propertyIndexOf. it could use the new StringIntMap, but that one isn't available yet.

Reply:
ok